web/src/main/webapp/resources/js/atsy-settings.js

```diff
@@ -100,4 +100,4 @@ window.positionsEvents = {
         var options = getOptions('question.delete.position.js', 'selected.position.not.found.js', row, container);
         bootbox.dialog(options);
     }
-};
```

Comment:
If nothing changed here,this could be reverted, so this change won't show up when looking at the merged code to master.